mantidproject/mantid
# ISIS Reflectometry: Ensure batch tabs numbers are unique

**Description of work.**
Previously Batch tabs could have multiple of the same name, due to utilising the count as the name creation, however, an incrementing counter is now used.



**To test:**
- Open Workbench
- Launch Interfaces->ISIS Reflectometry
- Create multiple batches by going to menu Batch->New Batch
- Remove a few of them randomly
- Create more batches
- No batches should have duplicated names



Fixes #xxxx. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
